x-pack/plugins/cases/public/components/create/case_owner_selection.test.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { act, waitFor } from '@testing-library/react';
+
+import { useForm, Form, FormHook } from '../../common/shared_imports';
+import { CaseOwnerSelection } from './case_owner_selection';
+import { schema, FormProps } from './schema';
+
+const OBSERVABILITY = 'observability';
+const SECURITY_SOLUTION = 'securitySolution';
+
+describe('Case Owner Selection', () => {
+  let globalForm: FormHook;
+
+  const MockHookWrapperComponent: React.FC = ({ children }) => {
+    const { form } = useForm<FormProps>({
+      defaultValue: { selectedOwner: '' },
+      schema: {
+        selectedOwner: schema.selectedOwner,
+      },
+    });
+
+    globalForm = form;
+
+    return <Form form={form}>{children}</Form>;
+  };
+
+  beforeEach(() => {
+    jest.resetAllMocks();
```

Comment:
nit: I think is better to use `jest.clearAllMocks`. The `jest.resetAllMocks` resets the implementation of the mock functions with a new `jest.fn()` and maybe that is not the desired behavior especially if you mock `useKibana`.

Reply:
changed 🍻